src/pages/home/report/ReportActionCompose.js

```diff
@@ -177,8 +177,12 @@ class ReportActionCompose extends React.Component {
                                     )}
                                 </AttachmentPicker>
                                 <TextInputFocusable
+                                    noFocus={this.props.isSidebarShown && this.props.isSmallDevice}
```

Comment:
Maybe this would read better as `isFocusable` but the name of the component is "Text Input _Focusable_" after all. I'm not sure it makes sense to have that name and then the ability to block focus...